internal/cache/file/downloader/parallel_downloads_job.go

```diff
@@ -62,8 +62,9 @@ func (job *Job) downloadRange(ctx context.Context, dstWriter io.Writer, start, e
 
 	monitor.CaptureGCSReadMetrics(ctx, util.Parallel, end-start)
 
-	// Use of memory aligned buffer is not required if use of O_DIRECT is disabled.
-	if job.fileCacheConfig.EnableODirect {
+	// Use standard copy function if O_DIRECT is disabled and memory aligned
+	// buffer otherwise.
+	if !job.fileCacheConfig.EnableODirect {
```

Comment:
Can we get rid of the negation and reverse the if-else clauses so that it is easier to understand the flow. Also, please reduce nesting since the else-block is likely not needed here - https://testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html

Reply:
Discussed offline to keep it like because there is a statement(formatting error) below the `if-else` which is common to both `if` and `else` blocks.